hbase_sql/src/main/java/com/splicemachine/hbase/SpliceReplicationSinkChore.java

```diff
@@ -260,19 +264,29 @@ private void updateProgress() throws IOException, KeeperException, InterruptedEx
                     SpliceLogUtils.info(LOG, "Checking snapshot taken at %d", timestamp);
                 //}
                 CellScanner s = r.cellScanner();
+                long ts = -1;
                 while (s.advance()) {
                     Cell cell = s.current();
-                    String walName = Bytes.toString(CellUtil.cloneQualifier(cell));
-                    Long position = Bytes.toLong(CellUtil.cloneValue(cell));
-                    if (replicationProgress.containsKey(walName)) {
-                        long appliedPosition = replicationProgress.get(walName);
+                    byte[] colName = CellUtil.cloneQualifier(cell);
+                    if(Arrays.equals(colName, HBaseConfiguration.REPLICATION_SNAPSHOT_TSCOL_BYTES)){
+                        ts = Bytes.toLong(CellUtil.cloneValue(cell));
                         //if (LOG.isDebugEnabled()) {
+                        SpliceLogUtils.info(LOG, "Process snapshot take at %s", new DateTime(ts).toString());
+                        //}
+                    }
+                    else {
+                        String walName = Bytes.toString(CellUtil.cloneQualifier(cell));
```

Comment:
The call to CellUtil.cloneQualifier(cell) here seems redundant, should we just use colName?